consumer-configuration/README.adoc

```diff
@@ -0,0 +1,347 @@
+* {base-url}{rhoas-cli-url}[_Getting started with the `rhoas` CLI_^]
+* {base-url-cli}{rhoas-cli-ref-url}[_CLI command reference (rhoas)_^]
+
+[id="ref-supported-consumer-group-properties_{context}"]
+== Consumer group properties presented in {product}
+
+[role="_abstract"]
+The following consumer group properties are presented in {product}.
+Consumer group properties are used for monitoring in the {product} web console and are not editable.
+
+=== Consumer groups
+
+Consumer Group ID::
+The consumer group ID is the unique identifier for the consumer group within the cluster. This is part of the consumer configuration for the application client.
+
+Active Members::
+Active members shows the number of consumers in the group that are assigned to a topic partition in the Kafka instance.
+If you are viewing information on consumer groups for a topic, these are the active members for the topic.
+
+Partitions with lag::
+Partitions with lag shows the number of partitions where the assigned consumer has not caught up with the last message in the partition.
+The lag reflects the position of the consumer offset in relation to the end of the partition log.
+
+=== Consumer offset positions
+
+When you click on a consumer group, you see the details of each member.
+
+Partition::
+The partition number for the topic.
+
+Client ID::
+The unique ID of the client application used to identify active consumers.
+If there is no client ID shown, the partition is not currently being consumed.
+
+Current offset::
+The current offset number for the consumer in the partition log. This is the position of the consumer in the partition and the latest read position.
+
+Log end offset::
+The current offset number for the producer in the partition log. This is the end of the log and the latest write position.
+
+Offset lag::
+The difference (delta) between the consumer and producer offset positions in the log.
+
+[id="con-reducing-consumer-lag_{context}"]
+== Consumer lag indicators
+
+[role="_abstract"]
+Consumer lag for a given consumer group indicates the delay between the last message added in a partition and the message currently being picked up by that consumer.
+The lag reflects the position of the consumer offset in relation to the end of the partition log.
+
+When you are reviewing consumer group properties in the {product} web console, look for the differences between *Current offset* and *Log end offset*.
+The difference shows as the *Offset lag* value.
+
+For applications that rely on the processing of (near) real-time data, it’s critical that consumer lag doesn't become too big.
+Suppose a topic streams 100 messages a second.
+A lag of 1000 messages between the producer offset (the topic partition head) and the last (current) offset the consumer has read means a 10-second delay.
+The offset lag shows a gap is opening up between the write and read positions.
+
+What can you do about the lag?
+Lag is often reduced by adding new consumers to a group.
```

Comment:
The question seems maybe a little too informal given the rest of the doc? If it were something that merited more emphasis, I guess fine. But I think unnecessary, plus the sentence that follows is passive and unclear whether it's the user or not who does it. So I'd just rewrite the paragraph as follows:

> To reduce lag, you typically add new consumers to a group. However, you can also increase ...